intro-HPC/ch_apptainer.tex

```diff
@@ -31,34 +31,35 @@ \section{Restrictions on image location}
 
 \section{Available filesystems}
 
-All HPC-UGent shared filesystems will be readily available in a Singularity container,
+All HPC-UGent shared filesystems will be readily available in an Apptainer/Singularity container,
 including the home, data and scratch filesystems, and they will be accessible via the
 familiar \lstinline|$VSC_HOME|, \lstinline|$VSC_DATA*| and \lstinline|$VSC_SCRATCH*| environment variables.
 
 \section{Singularity Images}
 
 \subsection{Creating Singularity images}
 
-Creating new Singularity images or converting Docker images, by default, requires admin privileges,
+Creating new Apptainer/Singularity images or converting Docker images, by default, requires admin privileges,
 which is obviously not available on the \hpcInfra infrastructure. However,
-if you use the \lstinline|--fakeroot| option, you can make new Singularity images or convert 
+if you use the \lstinline|--fakeroot| option, you can make new Apptainer/Singularity images or convert 
 Docker images. 
 
-When you make Singularity or convert Docker images you have some restrictions.
+Due to the nature of \lstinline|--fakeroot| option, we recommend to write your apptainer/singularity
+image to a globally writable location, like \lstinline|/tmp|, or \lstinline|/local| directories. 
+Once the images is created, you should move it to your desired destination.
```

Comment:
`Once the image is created` (singular `image`)

Reply:
fixed